oops implemented

ashishprasher opened the pull request:
project structure changed to incorporate oops methodology

mahajanankur reviewed (commented):
These modifications look good to me. @mohanrana, could you kindly take a look and review the changes?

ashishprasher committed:
oops implemented
easy-linter integrated
access token & refresh token logic removed from end user's scope
and other optimizations

mahajanankur reviewed (approved):
The implementation has been reviewed and verified to ensure that everything meets the agreed-upon requirements.
